buddyboss/buddyboss-platform
# PROD-734 #2988 Fixed mentioning in blog post comments doesn't send notifications

### All Submissions:
https://buddyboss.atlassian.net/browse/PROD-5126






### Changes proposed in this Pull Request:



Fixes #2988.

### How to test the changes in this Pull Request:

1. Enable the wordpress post and comments settings in the buddyboss settings > activities under "Posts in Activity Feeds"
2. Create a new sample blog
3. Comment and mention any user in the blog comment
4. View as the user you tagged, Notification should be there as well as an Email for the mention

### Proof Screenshots or Video

http://somup.com/cr6UXi3EJr

### Other information:




### Changelog entry

> Mentioning in blog post comments sends notifications
